packages/relay/tests/lib/eth.spec.ts

```diff
@@ -2219,6 +2231,38 @@ describe('Eth calls using MirrorNode', async function () {
       expect(hasError).to.be.true;
     });
 
+    it('eth_getStorageAt should throw a predefined RESOURCE_NOT_FOUND when block is null', async function () {
+      defaultDetailedContractResultsNullStateChange
+      mock.onGet(`blocks/${blockNumber}`).reply(200, defaultBlock);
+      mock.onGet(`contracts/${contractAddress1}/results?timestamp=lte:${defaultBlock.timestamp.to}&limit=1&order=desc`).reply(200, defaultContractResults);
+      mock.onGet(`contracts/${contractAddress1}/results/${contractTimestamp1}`).reply(200, defaultDetailedContractResultsNullStateChange);
+      let hasError = false;
+      try {
+        await ethImpl.getStorageAt(contractAddress1, defaultDetailedContractResults.state_changes[0].slot, EthImpl.numberTo0x(blockNumber));
+      } catch (e: any) {
+        hasError = true;
+        expect(e.code).to.equal(-32001);
```

Comment:
Is this magic number something I should just know were I working with typescript?

Reply:
It's the [RESOURCE_NOT_FOUND error code](https://github.com/hashgraph/hedera-json-rpc-relay/blob/main/packages/relay/src/lib/errors/JsonRpcError.ts#L118)

Reply:
Might be good to use `predefined.RESOURCE_NOT_FOUND.code` in the check instead

Reply:
done